pkg/services/services.go

```diff
@@ -136,33 +136,31 @@ func hybridOverlayConfiguration(vxlanPort string, debug bool) servicescm.Service
 
 // kubeProxyConfiguration returns the Service definition for kube-proxy
 func kubeProxyConfiguration(debug bool) servicescm.Service {
-	sanitizedSubnetAnnotation := strings.ReplaceAll(nodeconfig.HybridOverlaySubnet, ".", "\\.")
-	cmd := fmt.Sprintf("%s -log-file=%s %s --windows-service --proxy-mode=kernelspace --feature-gates=WinOverlay=true,WinDSR=true "+
-		"--hostname-override=NODE_NAME --kubeconfig=%s --cluster-cidr=NODE_SUBNET "+
-		"--network-name=%s --source-vip=ENDPOINT_IP --enable-dsr=true", windows.KubeLogRunnerPath, windows.KubeProxyLog,
-		windows.KubeProxyPath, windows.KubeconfigPath, windows.OVNKubeOverlayNetwork)
+	cmd := fmt.Sprintf("%s -log-file=%s %s --source-vip=ENDPOINT_IP --config %s", windows.KubeLogRunnerPath, windows.KubeProxyLog,
```

Comment:
is the `--source-vip=ENDPOINT_IP `  replacement needed? looks like it is happening in the powershell script already:

```
  sourceVip: $ipv4Addres
```